_data/companies.yml

```diff
@@ -1041,6 +1041,13 @@
   events: Restricted
   last_update: 2020-03-12
 
+- name: OpenCorporates
+  wfh: Optional
```

Comment:
Since it's required as of Monday, I'd go ahead and mark it as required now unless you're planning on submitting another PR over the weekend. Which way would you like to go?